File: src/app/model/gamerules.ts
export class GameRules {
  id: number;
  smallBlind: number;
  bigBlind: number;
  playDelay: number;
  startingChips: number;
  maxPlayerCount: number;
  minLevel: number;
  maxLevel: number;

  static create() {
    return {
      id: 0,
      smallBlind: 10,
      bigBlind: 20,
      playDelay: 30,
      startingChips: 1000,
      maxPlayerCount: 6,
      minLevel: 1,
      maxLevel: 50
    };
  }
}

File: mobile-client/app/src/main/java/be/kdg/mobile_client/room/model/GameRules.java
package be.kdg.mobile_client.room.model;

import lombok.Getter;

/**
 * The game rules that are applied to a specific room.
 */
@Getter
public class GameRules {
    private int smallBlind;
    private int bigBlind;
    private int playDelay;
    private int startingChips;
    private int maxPlayerCount;
    private int minLevel;
    private int maxLevel;
}
